internal/confmapprovider/discovery/bundle/bundle.d/receivers/smartagent-collectd-mysql.discovery.yaml

```diff
@@ -3,7 +3,7 @@
 # All changes must be made to associated .tmpl file before running 'make bundle.d'. #
 #####################################################################################
 smartagent/collectd/mysql:
-  enabled: true
+  enabled: false
```

Comment:
This shouldn't be disabled, and all changes should be done to the .tmpl as the header explains.